plugin/pki/path_roles_test.go

```diff
@@ -158,17 +137,4 @@ func TestRoleValidate(t *testing.T) {
 	if entry.StoreBy != storeByCNString {
 		t.Fatalf("Expecting store_by parameter will be set to %s", storeBySerialString)
```

Comment:
Not introduced here but I think this error message should be saying it expected `store_by` to be set to `storeByCNString`

Reply:
Fixed